index.js

```diff
@@ -0,0 +1,134 @@
+const {Client} = require("pg");
+
+const client = new Client("postgres://localhost:5432/costume_shop_db_dev");
+client.on("error", (error) => {
+    console.error(error.stack())
+})
+
+const createTables = async () => {
+    await client.query(`
+        DROP TABLE costumes;    
```

Comment:
This will run the first time even if the table doesn't exist yet

```suggestion
        DROP TABLE IF EXISTS costumes;    
```

Reply:
Yes. I think `IF EXISTS` is included in this on a more recent branch (`pool-pg` for sure).